scrapers/FragileSlave.yml

```diff
@@ -0,0 +1,46 @@
+name: "Fragile Slave"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fragileslave.com/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[@class="videodetails"]/p[@class="date"]
+    scene:
+      Title:
+        selector: //div[@class="pagetitle"]//h1/text()
+      Details:
+        selector: //div[@class="videocontent"]/p/text()
+      Date:
+        selector: $details[1]/text()
+        postProcess:
+          - replace:
+              - regex: (\s*\|.*)
+                with:
+              - regex: (\d{2})\/(\d{2})\/(\d{4})
+                with: $3-$1-$2
+          - parseDate: 2020-02-01
```

Comment:
The selector matches the commas as well. Adding a `[1]` at the  end gets us the correct text only
The parseDate should be `          - parseDate: 2006-01-02` 
Tbf even the second regex is not needed
```yaml
      Date:
        selector: $details[1]/text()[1]
        postProcess:
          - replace:
              - regex: (\s*\|.*)
                with:
          - parseDate: 01/02/2006
 ```
is enough

Reply:
corrected, thank you for the date suggestion